distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -0,0 +1,33 @@
+priority "grant";
+
+// Double forward slashes "//" indicate comments and have no effect on policies.
+
+
+//                        Content Directory Monitor (CDM) Permissions
+// Adding required permissions for a new Content Directory Monitor.
+//
+// If the directory being monitored is located in DDFs home directory, then adding these permissions
+// is not required and these instructions can be ignored.
+//
+// Configuring CDM requires adding read and write permissions to the directory being monitored. For
+// each configured CDM, the following 2 permissions are required:
+//   1. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+//   2. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read, write";
+// Replacing <DIRECTORY_PATH> with the path of the directory being monitored.
+//
+// Line 1 gives the CDM the permissions to read from the monitored directory path. Line 2 gives the
+// CDM the permissions to recursively read and write from the monitored directory path, specified
+// by the directory path's suffix "${/}-".
```

Comment:
Would you add some clarification about when I would need to allow read and write and when I would only need to allow write?

Reply:
When I originally did this I had ~4 sections required to add permissions for the directory being configured, which resulted in a lot of instructions and even more confusing instructions. So instead I collapsed everything needed into 1 section and now only 2 need to be added. This means, however, that some bundles will have permissions it does not need. However, since it only affects the directory, I do not think having extra permissions is bad in order to avoid a lot more complexity for the admin.

Reply:
I misread the docs before. Sorry for the question. LGTM